components/Header/index.js

```diff
@@ -1,3 +1,21 @@
-import Header from './Header';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import DefaultHeader from './DefaultHeader';
+import TransparentHeader from './TransparentHeader';
+
+const Header = ({ type }) => {
+  switch (type) {
+    case 'transparent':
+      return <TransparentHeader />;
+    default:
+      return <DefaultHeader />;
+  }
+};
+
+Header.propTypes = {
+  type: PropTypes.oneOf(['transparent']),
+};
+Header.defaultProps = {};
```

Comment:
Add a default option to the `type` property for clarity
```suggestion
Header.defaultProps = {
     type: "default"
};
```